app/src/main/java/net/bible/service/format/osistohtml/taghandler/MyNoteMarker.kt

```diff
@@ -40,7 +41,12 @@ class MyNoteMarker(private val parameters: OsisToHtmlParameters,
     override fun start(attr: Attributes) {
         if (parameters.isShowMyNotes) {
             if (myNoteVerses.contains(verseInfo.currentVerseNo)) {
-                writer.write("<img src='file:///android_asset/images/pencil16x16.png' class='myNoteImg'/>")
+                var tag: String = "<a href='%s:%s'><img src='%s' class='myNoteImg'/></a>".format(
+                    Constants.MYNOTE_PROTOCOL,
+                    parameters.basisRef.osisID + verseInfo.currentVerseNo,
+                    "file:///android_asset/images/pencil16x16.png"
+                )
```

Comment:
I like more kotlin's string intrepolation syntax

Reply:
I'll use that next time then